en/frames_multicopter/holybro_x500V2_pixhawk5x.md

```diff
@@ -0,0 +1,223 @@
+# Holybro X500 V2 (Pixhawk 5X Build)
+
+This topic provides full instructions for building the kit and configuring PX4 using *QGroundControl*.
+
```

Comment:
Add a paragraph here explaining what makes this kit special. 

In particular what's in it - that's the fact it has the frame, motors, power distribution board, so all you need are autopilot electronics - FC, gps/compass, receiver, telemetry radio, RC receiver. You will also need battery, transmitter.

Another point being that this kit is easy to adapt to different flight controllers.

Reply:
Added.